src/vecdb/vectorizer_service.rs

```diff
@@ -170,8 +203,11 @@ impl FileVectorizerService {
         cooldown_secs: u64,
         splitter_window_size: usize,
         splitter_soft_limit: usize,
-        embedding_model_name: String,
+
+        model_name: String,
         api_key: String,
+        cloud_name: String,
+        endpoint_template: String,
```

Comment:
global_context? 

To quit if caps changes significantly.

cloud_name is not a thing.